scripts/build.sh

```diff
@@ -727,6 +729,7 @@ if [ "$REMOVE_AMAZON" ]; then
     echo "Remove Amazon Appstore"
     find "${PRODUCT_MNT:?}"/{etc/permissions,etc/sysconfig,framework,priv-app} 2>/dev/null | grep -e amazon -e venezia | sudo xargs rm -rf
     find "${SYSTEM_EXT_MNT:?}"/{etc/*permissions,framework,priv-app} 2>/dev/null | grep -e amazon -e venezia | sudo xargs rm -rf
+    rm -rf "$WORK_DIR/wsa/$ARCH/apex"
```

Comment:
Do not remove any [apex](https://source.android.com/docs/core/ota/apex), which will cause system exceptions.